pom.xml

```
@@ -58,18 +58,20 @@
 
     <dependencies>
         <dependency>
-            <groupId>com.microsoft.azure</groupId>
-            <artifactId>azure</artifactId>
-            <version>1.9.1-SNAPSHOT</version>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-identity</artifactId>
+            <version>1.3.1</version>
         </dependency>
         <dependency>
-            <groupId>com.microsoft.azure</groupId>
-            <artifactId>azure-mgmt-keyvault</artifactId>
-            <version>1.11.1</version>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-core</artifactId>
+            <version>1.17.0</version>

```

There's a new version for this library.
```suggestion
            <version>1.21.0</version>
```